cachito/web/migrations/env.py

```diff
@@ -23,7 +24,12 @@
 # target_metadata = mymodel.Base.metadata
 from flask import current_app
 
-config.set_main_option("sqlalchemy.url", current_app.config.get("SQLALCHEMY_DATABASE_URI"))
+database_uri = (
+    ""
+    if current_app.config.get("SQLALCHEMY_DATABASE_URI") is None
+    else str(current_app.config.get("SQLALCHEMY_DATABASE_URI"))
```

Comment:
It could be just this
```suggestion
    current_app.config.get("SQLALCHEMY_DATABASE_URI", "")
```

but will this code even work if URL is undefined? Shouldn't we fail instead?

Reply:
Good point. What I was going to suggest next was:
1. Try to access SQLALCHEMY_DATABASE_URI. Don't handle a potential KeyError and let it fail
2. If the value of SQLALCHEMY_DATABASE_URI is None, raise a ValueError
3. Do the config.set_main_option, while casting the value of SQLALCHEMY_DATABASE_URI to string

FYI, the checks for 1 and 2 appear to be handled ahead of time by: https://github.com/containerbuildsystem/cachito/blob/f59adc16eba2166d43345a4c4da71b80fee261ce/cachito/web/config.py#L128

Reply:
I vote for 2.